chalice/deploy/planner.py

```diff
@@ -59,15 +60,51 @@ def get_remote_model(self, resource):
                            role_arn=role['Arn'])
 
 
+class Sweeper(object):
```

Comment:
`Sweeper` is still a little vague to me. Mainly I first interpreted is as an abstraction that sweeps through all of the resources, deleting them or sweeping through the two deployments (the previous one and the current planned one) and deletes the ones accounted for. So maybe make it something more specific like `UnreferencedResourceSweeper`?

Reply:
If the `Sweeper` name is confusing, it might be better to just call it `UnreferencedResourcePlanner`?

Reply:
Yeah that's better.